share/OpenMS/CHEMISTRY/Enzymes.xml

```diff
@@ -84,6 +89,10 @@
       <ITEM name="PSIid" value="MS:1001309" type="string" />
       <ITEM name="XTANDEMid" value="[K]|{P}" type="string" />
       <ITEM name="OMSSAid" value="5" type="int" />
+      <ITEM name="COMETid" value="3" type="int" />
+    </NODE>
+    <NODE name="Lys-N">
+    <ITEM name="COMETid" value="4" type="int" />
```

Comment:
great idea. could you also add theother fields: 
 RegEx and RegExDescription, NTermGain, PSIid, 

I think it might be also possible to give an XTANDEMid